tfquaternion/tfquaternion.py

```diff
@@ -163,8 +163,42 @@ def rotate_vector_by_quaternion(q, v, q_ndims=None, v_ndims=None):
         v = tf.expand_dims(v, axis=0) + tf.zeros_like(q_xyz)
     q_xyz += tf.zeros_like(v)
     v += tf.zeros_like(q_xyz)
-    t = 2 * tf.cross(q_xyz, v)
-    return v + tf.expand_dims(w, axis=-1) * t + tf.cross(q_xyz, t)
+    t = 2 * tf.linalg.cross(q_xyz, v)
+    return v + tf.expand_dims(w, axis=-1) * t + tf.linalg.cross(q_xyz, t)
+
+
+def quaternion_between_3d(u, v, epsilion=1e-6, dtype=tf.float32):
```

Comment:
Could you rename this function to be a bit more descriptive? E.g. `get_rotation_quaternion_from_u_to_v`